package.json

```diff
@@ -37,7 +37,8 @@
     "react-art": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-native-web": "^0.6.0",
-    "react-scripts": "^1.1.1"
+    "react-scripts": "^1.1.1",
+    "svgs": "^3.2.1"
```

Comment:
Seems like the package-lock.json shouldn’t change so much here. Which node/npm version are you using? We should stick with the current LTS node@8.x/npm@5.x since newer versions of npm will rebuild the package-lock.json and potentially cause incompapatibilites on travis.

Reply:
Gotcha, i'll switch versions